starters/next-wordpress-starter/lib/Menus.js

```diff
@@ -0,0 +1,35 @@
+import { gql } from "@pantheon-systems/wordpress-kit";
+import { client } from "./WordpressClient";
+
+export async function getFooterMenu() {
+  const query = gql`
+    query FooterMenuQuery {
+      menus(where: { slug: "Example Menu" }) {
+        edges {
+          node {
+            id
+            menuItems {
+              edges {
+                node {
+                  id
+                  uri
+                  label
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  `;
```

Comment:
I think we can get even more specific with the query to avoid the flatMap
```suggestion
    query ExampleMenuQuery {
      menu(idType: NAME, id: "Example Menu") {
        menuItems {
          nodes {
            id
            label
            path
          }
        }
      }
    }
  `;
```

Reply:
Good catch! Thank you